build.proj

```diff
@@ -310,7 +310,7 @@
   <!-- Run Dependecy Analyzer -->
   <Target Name="DependencyAnalysis">
     <Message Importance="high" Text="Running Dependency Analysis..." />
-    <Exec Command="$(MSBuildProjectDirectory)\src\Package\StaticAnalysis.exe $(MSBuildProjectDirectory)\src\Package\$(Configuration) $(MSBuildProjectDirectory)\src\Package"/>
+    <!--<Exec Command="$(MSBuildProjectDirectory)\src\Package\StaticAnalysis.exe $(MSBuildProjectDirectory)\src\Package\$(Configuration) $(MSBuildProjectDirectory)\src\Package"/>-->
```

Comment:
@securityadmin please revert this change; StaticAnalysis should never be disabled